prow/jobs/kyma-project/runtime-watcher/runtime-watcher.yaml

```diff
@@ -81,7 +81,7 @@ postsubmits:
               - "--config=/config/kaniko-build-config.yaml"
               - "--context=runtime-watcher"
               - "--dockerfile=Dockerfile"
-              - "--tag=latest"
+              - "--tag={{.Env \"PULL_BASE_REF\" }}"
```

Comment:
Change it to `$(PULL_BASE_REF)`. Should work the same and be more clean

Reply:
@Ressetkk I don't see syntax you propose in the project ATM. Are you sure it would work?

Reply:
https://github.com/kyma-project/test-infra/blob/main/prow/jobs/kyma/periodics.yaml look at this file. Should work.

Reply:
done.